api/models/log_event.py

```diff
@@ -29,7 +30,7 @@ class LogEvent(models.Model):
     """
 
     comment = models.TextField()
-    date_time = models.DateTimeField(auto_now_add=True)
+    date_time = models.DateTimeField(default=timezone.now)
```

Comment:
Note: https://docs.djangoproject.com/en/4.2/ref/models/fields/#django.db.models.DateField.auto_now_add

Reply:
I see, yes. Since we now want to modify `date_time`, we do have to make this change 👍🏻